javascript/javascript1/week4/homework/voiceA.js

```diff
@@ -0,0 +1,112 @@
+let nameSave = "";
+let todos = [];
+const suffixFirst = "hello my name is ";
+const suffixSecond = "what is my name";
+const suffixThird = "add";
+const prefixThird = "to my todo";
+const suffixFourth = "remove";
+const prefixFourth = "from my todo";
+const suffixFifth = "what is on my to-do";
+const suffixSixth = "what day is it today";
+const suffixSeventh = "what is";
```

Comment:
Consider better naming. suffixSecond, Third and so forth doesn't convey exactly what this variable is related to